camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -178,53 +189,76 @@
                 </b-modal>
               </b-col>
               <b-col class="align-self-center">
-              <span v-if="task.assignee">
-                <div v-if="editAssignee" class="cft-user-edit">
-                  <div class="cft-assignee-change-box">
-                    <v-select
-                      @search="fetchOptions"
-                      :options="autoUserList"
-                      v-model="userSelected"
-                      placeholder="Search by Lastname"
-                      class="assignee-align float-left"
-                    />
-                    <i
-                      @click="onSetassignee"
-                      class="fa fa-check assignee-tickmark-icon icon-border"
-                    ></i>
-                    <i
-                      @click="toggleassignee"
-                      class="fa fa-times assignee-cancel-icon icon-border"
-                    ></i>
+                <span v-if="task.assignee">
+                  <div v-if="editAssignee" class="cft-user-edit">
+                    <div class="cft-assignee-change-box">
+                      <v-select
+                        :label="selectSearchType"
+                        :options="reviewerUserList"
+                        v-model="userSelected"
+                        placeholder="Search by Lastname"
+                        class="assignee-align float-left"
+                      />
+                      <i
+                        @click="onSetassignee"
+                        class="fa fa-check assignee-tickmark-icon icon-border"
+                      ></i>
+                      <i
+                        @click="toggleassignee"
+                        class="fa fa-times assignee-cancel-icon icon-border"
+                      ></i>
+                      <b-dropdown
```

Comment:
can you make this icon lil bigger to match it with the 'tick` and 'close' icon

Reply:
Made the filter icon larger

![image](https://user-images.githubusercontent.com/70306694/130419969-1a457296-63bb-4833-95bc-e3b407defc29.png)